### DISCUSSION TITLE
Adding your project to the landscape

### DISCUSSION THREAD

**Discussion by nitishm** [cdfoundation/cdf-landscape#190] [Q&A]
Is there a special approval process that one would need to go through to add their project to the landscape. Is this limited to projects under the LF umbrella?

I would like to nominate https://github.com/Azure/orkestra for the CD category (or whatever category the team decides is right).

**Comment by markjacksonfishing** [comment]
As noted [here](https://github.com/cdfoundation/cdf-landscape) 
```
This landscape is intended as a map to explore various tools and services that support a continuous delivery-oriented software environment from individual process steps to full pipeline orchestration, and also shows the member companies of the LF Continuous Delivery Foundation.
```
To my knowledge, Azure is not a member company nor any of its affiliates.  Now this does not mean you cannot open a PR to add. It will be reviewed.

Hopefully that answers your question and thank you kindly for checking.

**Reply by nitishm** [reply]
That helps - let me create the PR in that case.
